examples/ipv6/rpl-tsch/project-conf.h

```diff
@@ -164,4 +164,9 @@
 
 #endif /* CONTIKI_TARGET_Z1 */
 
+#if CONTIKI_TARGET_CC2538DK || CONTIKI_TARGET_ZOUL || \
+    CONTIKI_TARGET_OPENMOTE_CC2538
+#define TSCH_CONF_HW_FRAME_FILTERING    0
+#endif /* CONTIKI_TARGET_CC2538DK || CONTIKI_TARGET_ZOUL */
```

Comment:
Minor issue, but maybe you could update (or remove) the comment to reflect all three defines